plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryIntegrationSmokeTest.java

```diff
@@ -61,6 +64,76 @@ public void testDescribeTable()
         assertEquals(actualColumns, expectedColumns);
     }
 
+    @Test(dataProvider = "createTableColumnTypes")
+    public void testCreateTableWithEveryType(String createType, String expectedType)
```

Comment:
add test checking that `CREATE TABLE` fails if table already exists and `CREATE TABLE IF NOT EXISTS` works fine as a noop.